dev/bots/prepare_package.dart

```diff
@@ -255,6 +264,8 @@ class ArchiveCreator {
   late File _outputFile;
```

Comment:
Not related to this change, but this ivar can be removed, it's only used in `createArchive()` and is never read again.
```dart
  Future<File> createArchive() async {
    assert(_version.isNotEmpty, 'Must run initializeRepo before createArchive');
    final File outputFile = File(path.join(outputDir.absolute.path, _archiveName));
    await _installMinGitIfNeeded();
    await _populateCaches();
    await _validate();
    await _archiveFiles(outputFile);
    return outputFile;
  }
```

I could make another PR to clean this up though.

Reply:
Good catch. Done.